automatic/totalcommander/update.ps1

```diff
@@ -4,33 +4,31 @@ $releases = 'https://www.ghisler.com/download.htm'
 
 function global:au_SearchReplace {
     @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            '(^\s*file\s*=\s*)(".*")'      = "`$1""`$toolsPath\$(Split-Path -Path $Latest.Url32 -Leaf)"""
```

Comment:
The call to `Get-RemoteFiles` sets `FileName32` and `FileName64`, so there is no need to do a split path on the URL itself.

```suggestion
            '(^\s*file\s*=\s*)(".*")'      = "`$1""`$toolsPath\$($Latest.FileName32)"""
```